mailing_list/templates/mailing_list/admin_index.html

```diff
@@ -45,11 +45,10 @@
     <div class="col-md-12">
           <h2>Mailing Lists for {{course_name}}</h2>
             <p class="caption">
-              This tool provides mailing lists that allow you to contact members
-              of your course using an email client (e.g. Outlook, Gmail, etc.)
-              as an alternative to the Canvas Inbox. You <b>must</b> send your
-              message from the email address that is set as your default in
-              Canvas or your message will not be delivered.
+              This tool provides mailing lists that allow you to contact members of your course using an email
```

Comment:
This textchange has already landed on develop, and probably doesn't need to be part of this PR.